apps/ios/LiveStream/LiveStream/API/Core/API.swift

```diff
@@ -19,6 +19,17 @@ import Foundation
 
 class API {
     static var shared = API()
+    
+    var environment: TwilioEnvironment {
+        if url.hasSuffix("dev.stage.twil.io") {
+            return .stage
+        } else if url.hasSuffix("dev.dev.twil.io") {
```

Comment:
@timmydoza this is safe right?

Reply:
I think the first `dev` is the serverless "environment", which is just any string that the customer gets to pick (it's not the same as the prod, dev, and stage environments that Twilio services run in). But the app doesn't allow the user to change this serverless "environment". 

So I wonder if `url.hasSuffix("stage.twil.io")` and `url.hasSuffix("dev.twil.io")` might be a touch safer, but I'm not sure if it really matters.

Reply:
That is what I had originally but found during testing that I broke prod, because the prod suffix is `dev.twil.io` also. So I had to take into consideration the extra `dev`. 

So I think we're all good then thanks.